### PR TITLE
feat(SVM): Added new field enableSvm(Default value - true) for linked project.

### PR DESCRIPTION
**Pull request by JaideepPalit**
> Added new field enableSvm(Default value - true) for linked project.
>  * Request/Response structure of get_projects_with_all_details, get_project_by_id, create_project, update_project for linked project changed

> * Which issue is this pull request belonging to and how is it solving it? (*Refer to issue here*)
> * Did you add or update any new dependencies that are required for your change?

Issue: 

 
> Have you implemented any additional tests? - No

### Checklist
Must:
- [X] All related issues are referenced in commit messages and in PR

**_Edit Project-_** 

**_Project Moderation Request-_** 

Signed-off-by: Jaideep Palit <jaideep.palit@siemens.com>

### CODE REVIEW

**Comment by JaideepPalit** [comment]
Updated field names to `enableSvm` and `ProjectProjectRelationskip`

**Comment by smrutis1** [comment]
Reviewed it.
1. Please update the script numbering and readme.
2. Got this error while executing one of the script `045`. May be a `None` check is necessary.
image

File: scripts/migrations/044_migrate_project_linked_project_relation.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        log['updatedProjects'].append(updatedProject)
+        if not DRY_RUN:
+            db.save(project)
+    resultFile = open('044_migrate_project_linked_project_relation.log', 'w')
+    json.dump(log, resultFile, indent = 4, sort_keys = True)
+    resultFile.close()
+
+    print '\n'
+    print '------------------------------------------'
+    print 'Total Projects with linkedProjects'
```
Comment (smrutis1) [inline scripts/migrations/044_migrate_project_linked_project_relation.py:82]: Please append the count, if it was intended to.

**Review by smrutis1 — CHANGES_REQUESTED** [review]
Some observations

**Comment by JaideepPalit** [comment]
Thanks for review.
1. Changed
2. Fixed

**Comment by smrutis1** [comment]
Have tested as per the mentioned steps. Working as expected.

Note: As there is structure change in API, the existing clients might break.

**Review by smrutis1 — APPROVED** [review]
Code looks good to me.

**Comment by JaideepPalit** [comment]
(from discussion) - Added backward compatibility for `Create/update project with linked project`.
How to Test - 
   - Test Rest API -create_project, update_project with/without linked project
       - Old Structure for linked project
           ```
           "linkedProjects": {
                 "cf46ecaf1f2f9295xxxxxxxx" :  "UNKNOWN",
                 "cf46ecaf1f2f92xxxxxxxxxx": "UNKNOWN",
                 "cf46ecaf1f2f9xxxxxxxxxxx":   "UNKNOWN"
           }
          ```
     - New Structure  for linked Project
         ```
         "linkedProjects": {
                 "cf46ecaf1f2f9xxxxxxxxxxxx" :  {
                        "enableSvm": "false",
                        "projectRelationship": "CONTAINED"
                 },
                 "cf46ecaf1f2f92xxxxxxxxx": {
                        "enableSvm": "true",
                        "projectRelationship": "CONTAINED"
                 },
                 "cf46ecaf1f2f929xxxxxxxx":   {
                        "projectRelationship": "CONTAINED"
                 }
         }
         ```

**Pull request merged**